api/src/pcapi/routes/serialization/finance_serialize.py

```diff
@@ -53,3 +64,72 @@ def from_orm(cls, invoice: finance_models.Invoice) -> "InvoiceResponseModel":
 
 class InvoiceListResponseModel(BaseModel):
     __root__: list[InvoiceResponseModel]
+
+
+class LinkedVenues(BaseModel):
+    """A venue that is already linked to a bank account."""
+
+    id: int
+    commonName: str
+
+    class Config:
+        orm_mode = True
+
+    @classmethod
+    def from_orm(cls, venue: "offerers_models.Venue") -> "LinkedVenues":
+        venue.commonName = venue.common_name
+        return super().from_orm(venue)
+
+
+class ManagedVenues(BaseModel):
+    id: int
+    commonName: str
+    siret: str
+    bankAccountId: int | None
+
+    class Config:
+        orm_mode = True
+
+    @classmethod
+    def from_orm(cls, venue: "offerers_models.Venue") -> "ManagedVenues":
+        """
+        For each managed venue, we only want to display the current bank account
+        that is linked to it, if any.
+        """
+        if len(venue.bankAccountLinks) > 1:
+            logger.error("There should be one or no current bank account.")
```

Comment:
Suggestion (facultatif) : 
```suggestion
            logger.error("There should be one or no current bank account.", extra={"venue_id": venue.id})
```